service/service_setup.go

```diff
@@ -33,22 +33,23 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (svc *serviceCore) setupLogging(cfg config.Provider) {
+func (svc *serviceCore) setupLogging() {
 	if svc.log == nil {
-		logBuilder := ulog.Builder()
-		// load and configure logging
+		logBuilder := ulog.Builder().WithScope(svc.metrics)
```

Comment:
I agree. I did spend some time thinking about it, but I am not sure with the current state of things the implementation is going to be the cleanest. 

Right now, the thing that would fit best is to include a `disabled:` flag in config and allow user to specify that. However, I don't think that mixing and matching `enabled` and `disabled` in configuration is a good thing (I've found that the positive bools tend to work out better in the long run).

I think that a better fit for this is going to be the default configuration, which is currently in the works by @alsamylkin. Then, we can include `metrics: logging: enabled: true` or something like that, and allow to easily override in service config.

Basically it's a long way of saying I think we should wait to add that in the proper way. Thoughts?